package.json

```diff
@@ -46,9 +46,12 @@
 		"webpack-merge": "^4.1.1"
 	},
 	"engines": {
-		"node": ">=8"
+		"node": "9.3.0",
+		"npm": "5.5.1"
```

Comment:
npm comes with Node, but if we ever have to run `npm rebuild` yarn does not yet have that support.

That being said, I don't believe we need to specify its version since it comes bundled with whatever version of Node we are running.